app/helpers/dropdown_selection_helper.rb

```diff
@@ -6,4 +6,20 @@ def chosen_user
   def chosen_period
     params.dig(:metric, :period) || 4
   end
+
+  def chosen_from
+    params.dig(:metric, :from) || initial_from
+  end
+
+  def chosen_to
+    params.dig(:metric, :to) || initial_to
+  end
+
+  def initial_to
+    Time.zone.now.strftime('%Y-%m-%d')
+  end
+
+  def initial_from
+    (Time.zone.now - 28.days).strftime('%Y-%m-%d')
```

Comment:
```suggestion
    28.days.ago.strftime('%Y-%m-%d')
```

BTW why 28? I guess `2.weeks` might be a better default? IDK